Show file names in documentation (#12030)
supabase/supabase: apps/docs/pages/guides/getting-started/tutorials/with-vue-3.mdx

VITE_SUPABASE_ANON_KEY=YOUR_SUPABASE_ANON_KEY
```

> Now that we have the API credentials in place, let's create a helper file (`src/supabase.js`) to initialize the Supabase client. These variables will be exposed

Reviewer: ```suggestion
With the API credentials in place, create an `src/supabase.js` helper file to initialize the Supabase client. These variables are exposed
```

### Set up a Login component

> Let's set up a Vue component at `src/components/Auth.vue` to manage logins and sign ups. We'll use Magic Links, so users can sign in with their email without using passwords. 

Reviewer: ```suggestion
Set up an `src/components/Auth.vue` component to manage logins and sign ups. We'll use Magic Links, so users can sign in with their email without using passwords. 
```

### Account page

After a user is signed in we can allow them to edit their profile details and manage their account.

> Let's create a new component for that in `src/components/Account.vue`.

Reviewer: ```suggestion
Create a new `src/components/Account.vue` component to handle this.
```

### Create an upload widget

> Let's create an avatar for the user so that they can upload a profile photo. We can start by creating a new component at `src/components/Avatar.vue`:

Reviewer: ```suggestion
Create a new `src/components/Avatar.vue` component that allows users to upload profile photos:
```

### Add the new widget

> And then we can add the widget to the Account page at `src/components/Account.vue`:

Reviewer: ```suggestion
And then we can add the widget to the Account page in `src/components/Account.vue`:
```
